### DISCUSSION TITLE
INITIAL PROPOSAL: Parameters changes for `mint` module

### DISCUSSION THREAD

**Discussion by lezzokafka** [crypto-org-chain/chain-main#1027] [Ideas]
## Abstract
In response to the success of reaching and surpassing our initial target staking ratio of 30%, and in anticipation of the ongoing development of the Cronos POS chain - To ensure a reasonable and competitive incentive for staking delegation, thereby enhancing the network's security, we propose updating the following network parameters as follows: 
- `goal_bonded` from 0.3 to: 0.60
- `inflation_max` from 0.025 to: 0.037
- `Inflation_rate_change` from 0.006 to: 0.1

These proposed changes are intended to create a more substantial incentive for delegators to stake and demonstrate their commitment to the network. Along with an objective to encourage a higher staking ratio, contributing to a more secure network. 

## Background

Starting from 2023, there has been an upward trend in CRO staking ON Cronos POS Chain. The total staking amounts have risen from around 5 billion at the beginning of 2023 to the current level of 8.5 billion. With this overall increase in total delegation, there has been a direct impact on the staking APR, resulting in a decline from 12% to 6.5% at the time of writing.

<img width="500" alt="Screenshot 2024-01-16 at 2 20 38 PM" src="https://github.com/crypto-org-chain/chain-main/assets/51898510/b4ba9b1f-1917-40f7-977b-bf71c90f7cc1">

<img width="500" alt="Screenshot 2024-01-16 at 2 21 00 PM" src="https://github.com/crypto-org-chain/chain-main/assets/51898510/c42ea1cb-161c-4304-87db-02c3d6d9a131">

As a result of the increase in total staking amounts, the staking ratio (Total staking / Total supply) has increased from 18%  to 32%. 

The current staking ratio has now surpassed a critical threshold of 30%, triggering a reduction in the inflation rate (refer to "Technical details" on `goal_bonded`) and exerting an extra negative impact on the staking APR.

<img width="500" alt="Screenshot 2024-01-16 at 2 21 37 PM" src="https://github.com/crypto-org-chain/chain-main/assets/51898510/fc6f5901-e44f-4f8b-91c0-021668dcf7e8">

Going forward, one can expect continued surge in staking participation, particularly due to the wide adoption of  liquid staking use cases. An example case is Veno Finance, one of the leading liquid staking protocols on Cronos, which has successfully attracted over 110 million CRO in staking within just one year of its launch, indicative of an ongoing upward trend in staking activities.

In order to make staking returns more sustainable for CRO delegators, we are proposing a parameter update which will bring staking APR more in line with historical levels.

## Technical details

To ensure the sustainability and competitiveness of Cronos POS Chain, from a technical standpoint, we propose to update the following network parameters:

#### 1/ Update goal_bonded from 30% to 60%:

When the total staking amount is high as a percentage of total CRO supply, it means that a large share of the CRO holders is committed to participating in the security and governance of the blockchain protocol. In principle, the higher that percentage, the healthier the ecosystem.

The impact of the parameter “goal_bonded” (also called staking ratio) on the reward and the inflation rate can be summarized as follows:
- If the staking ratio is below the “goal_bonded”,' the inflation rate will increase until reaching a maximum upper bound(inflation_max); 
- If the staking ratio is equal to “goal_bonded”, the inflation rate will stay constant; 
- If the staking ratio is above the “goal_bonded”, it will decrease until reaching a minimum lower bound (inflation_min).

After successfully reaching and surpassing our initial target staking ratio of 30%, the current parameter settings have led us to the third scenario, resulting in an extra negative impact on overall staking rewards due to inflation drops. 

In light of this, we suggest increasing the target bonded ratio to 60%. Instead of maintaining the current downward pressure on the inflation rate, this adjustment will introduce an upward momentum for the inflation rate and create a more attractive incentive for delegators.

#### 2/  Update `inflation_max` from 0.025 to 0.037:

This parameter controls the upper bound of the maximum inflation rate. Since we have already reached the maximum inflation rate in the current configuration, along with adjusting the "goal_bonded," a higher ceiling for "inflation_max" is necessary to accommodate the increase in rewards required for achieving a higher APR. 

Under the current total supply of 26.4B, the proposed maximum inflation rate would offer additional upward headroom for the growth of the staking reward. As a reference, at the suggested maximum value of 0.037, we can achieve:
- 8.5% staking APR, at 35% of bounded ratio (9.24B staking);
- 7.5% staking APR, at 40% of bounded ratio (10.6B staking);
- 6.7% staking APR, at 45% of bounded ratio (11.8B staking);

These values provide a competitive staking reward to both delegators and validators, ensuring sufficient commission income to sustain their operations across various realistic scenarios.

#### 3/ Update inflation_rate_change from 0.006 to 0.1:

“Inflation_rate_change” represents the maximum percentage by which the inflation rate can change in a year. The magnitude of the rate of change of the inflation rate is controlled by an additional factor, which is the ratio between the current bonded ratio with the 'goal_bonded', the inflation rate is updated at the end of every block accordingly to the following formula:

```
Inflation Rate = min(max(params.InflationMin, currentInflation + ((1 - bondedRatio / params.GoalBonded) * params.InflationRateChange / params.BlocksPerYear)), params.InflationMax)
```

Or simply, when the inflation rate within the maximum and the minimum bounds
<img width="825" alt="Screenshot 2024-01-16 at 2 27 59 PM" src="https://github.com/crypto-org-chain/chain-main/assets/51898510/bfe501ca-9053-40d0-9b94-8f04fb3c43de">

This suggests that when the actual bonded ratio is at 40%, the final change in the inflation rate would be subject to a factor of 0.33 If the 'goal_bonded' is set to 60% (0.33 = 1 - 0.4/0.6). The factor will decrease as the bonded ratio increases, along with the growing number of staking. 
With the above, we propose updating the “inflation_rate_change” to 0.1, providing a more responsive rate for the inflation rate to be adjusted according to the actual bonded ratio and to encourage staking. 

## Testing 

The proposed change will be tested on the Croeseid testnet `testnet-croeseid-4` to ensure that the expected outcomes are achieved.

## Governance Votes 

The following items summarize the voting options and what it means for this proposal. 
- **YES**: You approve of the proposal on the changes to the proposed changes to the parameters to create more substantial incentive for delegators to stake, along with an objective to encourage a higher staking ratio, contributing to a more secure network. 
- **NO**: You disapprove of the proposal in its current form. Feel free to give feedback and suggestions on how to improve the proposal. 
- **NO (VETO)**: You veto the entire motivation for the proposal, and are strongly opposed to its implementation. 
- **ABSTAIN**: You are impartial to the outcome of the proposal. If the quorum is not met or less than 1/3 of participating voting power has voted No, the proposal fails and is considered to be vetoed. In this case, the proposal deposit is burned rather than being returned to respective depositors.

**Comment by zanglang** [comment]
Since it is clear we are now no longer bounded by the annual emissions goals and tuning inflation parameters to increase block rewards, why not temporarily disable the community tax parameter as well to reduce one more additional variable?

**Reply by AlBertTheVoid** [reply]
I agree. Considering the fact that the burning mechanism was introduced to limit the overflow inflation, which the team now wants to manage and deal with more comprehensively, the burn mechanism might be obsolete. Unless, there are other goals it also solves, such as community engagement, participation, etc.

**Reply by lezzokafka** [reply]
Hello both, thank you for the question and inputs.
Your comment on the community tax is valid. One potential way to think about this is that the inflation parameters are structurally sensitive and meant to be adjusted only very rarely, whereas the community tax was a tactical measure that the validators can use to adjust emission levels when they deviate temporarily from their expected level (as was the case last year when the community tax was set at its current level).

In the analogy of a water faucet, as described in the [blog post](https://blog.cronos-pos.org/p/fortifying-cronos-pos-network-security), the source of the flow represents the emission of CRO tokens, and this flow is set by the mint module and its parameters. On the other hand, the community tax acts as a mechanism that determines how this flow is distributed (15% to CRO community burn, 85% to delegators/validators). We believe these two mechanisms are distinct and can work in tandem.

Considering the continuous developments on the Cronos POS chain, we recognize the need to dynamically adapt to these changes. This is crucial to encourage a higher staking ratio, contributing to a more secure network. We hope this explanation makes sense, and we eagerly await further feedback on this matter.

**Comment by zeen69** [comment]
If this is the best proposal for the progress of the community I support it

**Reply by Gomez33** [reply]
[]()
